metrictank/src/main/java/com/expedia/metrics/metrictank/GaasKafkaValueDeserializer.java

```diff
@@ -0,0 +1,103 @@
+package com.expedia.metrics.metrictank;
+
+import com.expedia.metrics.MetricData;
+import com.expedia.metrics.MetricDefinition;
+import com.expedia.metrics.TagCollection;
+import org.apache.commons.codec.binary.Hex;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GaasKafkaValueDeserializer implements Deserializer {
```

Comment:
Not sure if this is tied to MetricTank generally or GaaS specifically. GaaS is Expedia-internal so I don't think we want to have a deserializer with that name.